pkg/cluster/etcd.go

```diff
@@ -6,5 +6,5 @@ import (
 )
 
 func init() {
-	managed.RegisterDriver(etcd.NewETCD())
+	managed.RegisterDriver(etcd.NewETCD(""))
```

Comment:
We are attempting to establish a k8s client connection in a package-level init function?

Reply:
Since remove between them and last push.